webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+
+import {Post} from 'mattermost-redux/types/posts';
+
+import Svgs from '../../constants/svgs';
+
+import {makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+
+export type Props = {
+    post?: Post,
+    theme: any,
+    creatorName: string,
+    useMilitaryTime: boolean,
+    meetingEmbedded: boolean,
+    actions: {
+        enrichMeetingJwt: (jwt: string) => any,
+        openJitsiMeeting: (post: Post | null, jwt: string | null) => void
```

Comment:
Why not use the `?` syntax?

Reply:
In the case of the "Post" because I explicitly ant to pass the post or the null value, passing `undefined` makes no sense. for the jwt, I think is acceptablo to pass the string or not passing it, but because i going to pass it anyway, I prefer to pass it as null when is null.